src/clients/apiClient.ts

```diff
@@ -1,12 +1,38 @@
 export const backendUrl = import.meta.env.VITE_BACKEND_URL;
 
+export interface NewSighting{
+    dateOfSighting: Date;
+    locationLatitude: number;
+    locationLongitude: number;
+    photoImageURL: string;
+    numberOfWhales: number;
+    description: string;
+    whaleSpecies: string;
+}
+
 export const checkBackendConnection = async (): Promise<boolean> => {
-  let response: Response;
-  try {
-    response = await fetch(`${backendUrl}/test`);
-  } catch {
-    return false;
-  }
-  
-  return response.ok;
+    let response: Response;
+    try {
+      response = await fetch(`${backendUrl}/test`);
+    } catch {
+      return false;
+    }
+    
+    return response.ok;
+}
+
+export async function createSighting(newSighting: NewSighting): Promise<Response> {
+    const response = await fetch(`https://${backendUrl}/createsighting`, {
```

Comment:
The URL will be take the controller route and then the endpoint route, something like `"${backendUrl}/sighting/create"`